common/lib/xmodule/xmodule/modulestore/tests/test_mixed_modulestore.py

```diff
@@ -2659,9 +2655,8 @@ def test_course_publish_signal_rerun_firing(self, default):
                 self.store.clone_course(course_key, dest_course_id, self.user_id)
                 signal_handler.send.assert_called_with('course_published', course_key=dest_course_id)
 
-    @patch('xmodule.tabs.CourseTab.from_json', side_effect=mock_tab_from_json)
```

Comment:
Why did we stop mocking this method in our tests? Is this what was returning the dict?

Reply:
Yep, that is it exactly